### PR TITLE
Add Min Active nomination to Staking pages

### PR DESCRIPTION
**Pull request by DrW3RK**

### CODE REVIEW

File: docs/learn/learn-nomination-pools.md [hunk 1]
```diff
@@ -307,6 +307,27 @@ the list of validators nominated by the pool, and so, in a way, you are trusting
...
+{{ kusama: __<RPC network="kusama" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>:__ :kusama }}
+. If you have less {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} than the minimum active nomination and still want to participate in staking,
+you can join the nomination pools.
+you can now stake on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} natively with
+just
+{{ polkadot: __<RPC network="polkadot" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={10000000000}/>__ :polkadot }}
+{{ kusama: __<RPC network="kusama" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={1666666650}/>__ :kusama }}
+in the nomination pools and earn staking rewards. For additional information, check out
+[this blog post](https://polkadot.network/blog/nomination-pools-are-live-stake-natively-with-just-1-dot/).
+Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more information.
```
Comment (filippoweb3) [inline docs/learn/learn-nomination-pools.md]:
```suggestion
The minimum amount required to become an active nominator and earn rewards is
{{ polkadot: __<RPC network="polkadot" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>__. :polkadot }}
{{ kusama: __<RPC network="kusama" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>__. :kusama }}
 If you have less {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} than the minimum active nomination and still want to participate in staking,
you can join the nomination pools.
You can now stake on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} natively with
just
{{ polkadot: __<RPC network="polkadot" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={10000000000}/>__ :polkadot }}
{{ kusama: __<RPC network="kusama" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={1666666650}/>__ :kusama }}
in the nomination pools and earn staking rewards. For additional information, see
[this blog post](https://polkadot.network/blog/nomination-pools-are-live-stake-natively-with-just-1-dot/).
Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more information.
```

File: docs/learn/learn-nominator.md [hunk 2]
```diff
@@ -22,6 +22,25 @@ help you get started.
...
+{{ kusama: __<RPC network="kusama" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>:__ :kusama }}
+. If you have less {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} than the minimum active nomination and still want to participate in staking,
+you can join the nomination pools.
+you can now stake on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} natively with
+just
+{{ polkadot: __<RPC network="polkadot" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={10000000000}/>__ :polkadot }}
+{{ kusama: __<RPC network="kusama" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={1666666650}/>__ :kusama }}
+in the nomination pools and earn staking rewards. For additional information, check out
+[this blog post](https://polkadot.network/blog/nomination-pools-are-live-stake-natively-with-just-1-dot/).
+Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more information.
```
Comment (filippoweb3) [inline docs/learn/learn-nominator.md]:
```suggestion
The minimum amount required to become an active nominator and earn rewards is
{{ polkadot: __<RPC network="polkadot" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>__. :polkadot }}
{{ kusama: __<RPC network="kusama" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>__. :kusama }}
 If you have less {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} than the minimum active nomination and still want to participate in staking,
you can join the nomination pools.
You can now stake on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} natively with
just
{{ polkadot: __<RPC network="polkadot" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={10000000000}/>__ :polkadot }}
{{ kusama: __<RPC network="kusama" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={1666666650}/>__ :kusama }}
in the nomination pools and earn staking rewards. For additional information, see
[this blog post](https://polkadot.network/blog/nomination-pools-are-live-stake-natively-with-just-1-dot/).
Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more information.
```

File: docs/learn/learn-staking.md [hunk 3]
```diff
@@ -22,11 +22,17 @@ help you get started.
...
+you can now stake on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} natively with
 just
-{{ polkadot: <RPC network="polkadot" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={10000000000}/> :polkadot }}
-{{ kusama: <RPC network="kusama" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={1666666650}/> :kusama }}
-and earn staking rewards. For additional information, check out
+{{ polkadot: __<RPC network="polkadot" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={10000000000}/>__ :polkadot }}
+{{ kusama: __<RPC network="kusama" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={1666666650}/>__ :kusama }}
+in the nomination pools and earn staking rewards. For additional information, check out
 [this blog post](https://polkadot.network/blog/nomination-pools-are-live-stake-natively-with-just-1-dot/).
 Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more information.
```
Comment (filippoweb3) [inline docs/learn/learn-staking.md]:
```suggestion
The minimum amount required to become an active nominator and earn rewards is
{{ polkadot: __<RPC network="polkadot" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>__. :polkadot }}
{{ kusama: __<RPC network="kusama" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>__. :kusama }}
 If you have less {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} than the minimum active nomination and still want to participate in staking,
you can join the nomination pools.
You can now stake on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} natively with
just
{{ polkadot: __<RPC network="polkadot" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={10000000000}/>__ :polkadot }}
{{ kusama: __<RPC network="kusama" path="query.nominationPools.minJoinBond" filter="humanReadable" defaultValue={1666666650}/>__ :kusama }}
in the nomination pools and earn staking rewards. For additional information, see
[this blog post](https://polkadot.network/blog/nomination-pools-are-live-stake-natively-with-just-1-dot/).
Check the wiki doc on [nomination pools](learn-nomination-pools.md) for more information.
```

**Review by filippoweb3 — APPROVED** [review]
lgtm, minor fixes to rendering

File: docs/learn/learn-staking.md [hunk 4]
```diff
@@ -22,11 +22,17 @@ help you get started.
 
 :::info Stake through Nomination Pools
 
-You can now stake on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} natively with
+
+The minimum amount of {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} required to become an active nominator and earn rewards is
```
Comment (profradha) [inline docs/learn/learn-staking.md]:
```suggestion
The minimum amount required to become an active nominator and earn rewards is
```

**Pull request merged**